react-blog/src/services/posts.ts

```diff
@@ -1,9 +1,19 @@
-import { api } from "./configs/http";
+// import { api } from "./configs/http";
+import axios from "axios";
 
 class PostsService {
   getAll() {
-    return api.get("/blog/posts/");
+    return axios.get("https://studapi.teachmeskills.by/blog/posts/?limit=100");
```

Comment:
Use our api
In that case u don't need to write 'https://studapi.teachmeskills.by/blog/' . Just '/posts/'
Also move query param limit into second arg like:
axios.get("https://studapi.teachmeskills.by/blog/posts/", {params: {limit: 100}});